feature_extractor/action.py

```diff
@@ -18,7 +21,7 @@
 SELL_POULTICE = 0
 SELL_RATION = 1
 N_MOVE = 4
-N_ATK_TARGET = N_NPC_CONSIDERED + N_ENEMY_CONSIDERED
+#N_ATK_TARGET = N_NPC_CONSIDERED + N_ENEMY_CONSIDERED
```

Comment:
remove?

Reply:
I havent' touched `action.py` yet. I'll get back to this when we have clear idea on how we want to do trans_action.

Reply:
added these back, as they are used in the below code. I'll deal with action.py later.